### PR TITLE
Remove any helm values for sonata manifests 

### PR DESCRIPTION
**Pull request by rgolangh**
As the sonataflow specific manifests are auto-generated and copied as-is to this repo, we can't include values in them

### CODE REVIEW

**Comment by rgolangh** [comment]
@gabriel-farache want me to do the same for move2kube?
@dmartinol remove from escalation as well if any?

**Comment by gabriel-farache** [comment]
@rgolangh yes please :)

**Comment by dmartinol** [comment]
> @gabriel-farache want me to do the same for move2kube? @dmartinol remove from escalation as well if any?

no escalation stuff ATM, in the upstream repo

File: charts/workflows/charts/move2kube/values.yaml [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
-  name: serverless-workflow-m2k # name of the workflow
-  image: quay.io/orchestrator/serverless-workflow-move2kube:latest # image containing the workflow
-  move2kubeURL: https://move2kube-route-sonataflow-infra.apps.cluster-8xfw.redhatworkshops.io # URL for move2kube instace
-  backstageNotificationURL: http://orchestrator-backstage.orchestrator/api/notifications/ # URL for backstage notification plugin
-kfunction:
-  name: m2k-save-transformation-func # name of the Knative function that save the transformation output to git
-  image: quay.io/orchestrator/serverless-workflow-m2k-kfunc:latest # image of the knative function
-instance:
-  name: move2kube # name of the move2kube instance deployment
-  image: quay.io/orchestrator/move2kube-ui:latest # image of the move2kube instance
```
Comment (gabriel-farache) [inline charts/workflows/charts/move2kube/values.yaml:14]: those manifests are not generated

File: charts/workflows/charts/move2kube/values.yaml [hunk 2]
```diff
@@ -1,15 +1,3 @@
-sshSecretName: sshkeys # name of the secret holding the ssh keys that will be used by move2kube resources
-brokerName: default # name of the broker used by Knative eventing resources
```
Comment (gabriel-farache) [inline charts/workflows/charts/move2kube/values.yaml:2]: not related to generated manifest

File: charts/workflows/charts/move2kube/values.yaml (hunk 1 above)
Comment (rgolangh) [reply charts/workflows/charts/move2kube/values.yaml:14]: sorry, reverting

File: charts/workflows/charts/move2kube/values.yaml [hunk 3]
```diff
@@ -1,15 +1,3 @@
...
-namespace: sonataflow-infra # name where install the resources
-workflow:
-  name: serverless-workflow-m2k # name of the workflow
-  image: quay.io/orchestrator/serverless-workflow-move2kube:latest # image containing the workflow
-  move2kubeURL: https://move2kube-route-sonataflow-infra.apps.cluster-8xfw.redhatworkshops.io # URL for move2kube instace
-  backstageNotificationURL: http://orchestrator-backstage.orchestrator/api/notifications/ # URL for backstage notification plugin
```
[the remark this answers is not in the document]
Comment (rgolangh) [reply charts/workflows/charts/move2kube/values.yaml:8]: @dmartinol idea with env from configmap

**Comment by masayag** [comment]
@rgolangh this seems to be obsolete. can we close?

**Pull request closed without merging**